jekyll/_cci2/deployment-integrations.md

```diff
@@ -348,3 +348,49 @@ workflows:
     git push --follow-tags
     ``` 
 5.  If tests passed, CircleCI will publish the package to npm automatically.
+
+## SSH
+
+To configure CircleCI
+to deploy your application over SSH,
+follow the steps below.
+
+1. Add the SSH key for the server
+to which you're deploying.
+For instructions, see the [Adding an SSH Key to CircleCI]({{ site.baseurl }}/2.0/add-ssh-key/) document.
+
+2. Add the SSH username and SSH hostname of your build VM as environment variables.
```

Comment:
hostname...-> hostname on the Settings > Environment Variables page of the CircleCI app.

Reply:
Same situation as my above comment.